mkdocs/docs/HPC/troubleshooting.md

```diff
@@ -291,6 +291,23 @@ ssh -vvv {{ userid }}@{{ loginnode }}
 and include the output of that command in the message.
 {% endif %}
 
+{% if site == gent %}
+## Issues reaching servers from HPC infrastructure
+
+If you have to reach license servers from {{ hpcinfra }} systems or you 
+have to directly load some database here, then it might not work (you will get 
+network connection timed out or network connection refused error). Our
+firewall rules are quite strict, we only allow outging ports 22 (SSH protocol),
```

Comment:
```suggestion
have to directly load some database here, then it might not work
(you will get errors like `Network connection timed out` or `Network connection refused`).

Our firewall rules are quite strict, we only allow outging ports 22 (SSH protocol),